frontend/src/webpages/chat/components/AgentActionListSideBar.tsx

```diff
@@ -16,13 +16,15 @@ import { useAuthContext } from "../../auth/AuthContext";
 import useChatComponent from "./hooks/useChatComponent";
 
 type TProps = {
-  isLoading: boolean;
+  isLoadingSummary: boolean;
```

Comment:
How about changing to only one loading state. When any of the action is triggered, disabled all and one spinner.